addons/assignGear/CfgEventHandlers.hpp

```diff
@@ -12,21 +12,21 @@ class Extended_PreInit_EventHandlers {
 
 class Extended_Init_EventHandlers {
```

Comment:
Should these be added through code using `CBA_fnc_addClassEventHandler` ? instead of pre-registering these event handlers?